src/user/contact.ts

```diff
@@ -479,8 +509,12 @@ export class Contact extends Accessory implements Sayable {
 
   /**
    * Contact gender
+   * > Tips: ContactGender is enum here. </br>
+   * - ContactGender.Unknown = 0</br>
```

Comment:
Please remove all numbers in the ENUM type.

`ContactGender.Unknown` is enough, the user should care about the number value about it, and also should never use the _magic number_.